libs/core/src/lib/product-switch/product-switch-body/product-switch-body.component.scss

```diff
@@ -0,0 +1,109 @@
+@import "~fundamental-styles/dist/product-switch";
+
+
+$product-switch-dnd-line-color: #0854A0;
+
+.cdk-drag-dragging {
+    z-index: 15;
+    background-color: white;
+    opacity: 0.8;
+    transition: box-shadow 200ms cubic-bezier(0, 0, 0.2, 1);
+    box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2),
+    0 2px 2px 0 rgba(0, 0, 0, 0.14),
+    0 1px 5px 0 rgba(0, 0, 0, 0.12);
+}
+
+.fd-dnd-placeholder {
+    top: 0;
+    left: 0;
+    position: absolute;
+    opacity: 0.3;
+    z-index: -1;
+}
+
+.fd-dnd-container {
+    position: relative;
+    padding: 0;
+    &.fd-dnd-on-drag {
+        background-color: white;
+        z-index: 15;
+        .fd-product-switch__item:active .fd-product-switch__icon {
+            color: #0854a0;
+        }
+
+        .fd-product-switch__item:active .fd-product-switch__title {
+            color: #32363a;
+        }
+
+        .fd-product-switch__item:active .fd-product-switch__subtitle {
+            color: #6a6d70;
+        }
+
+        &.selected {
+            background-color: white;
+        }
+
+        .fd-product-switch__item:active {
+            background-color: white;
+        }
+    }
+}
+
+.fd-product-switch__fullfill {
+    width: 500px;
+}
+
+.drop-area__line {
+    z-index: 10;
+    position: absolute;
+    background: $product-switch-dnd-line-color;
+    &--vertical {
+        top: 0;
+        height: 100%;
+        width: 2px;
+
+        &.after {
+            right: -0.25rem;
+        }
+
+        &.before {
+            left: -0.25rem;
+        }
+
+        &:before {
+            content: '';
+            position: absolute;
+            left: -3px;
+            top: -7px;
+            border-radius: 50%;
+            width: 0.5rem;
+            height: 0.5rem;
+            border: 0.125rem solid $product-switch-dnd-line-color;
+        }
+
+        &:after {
+            content: '';
+            position: absolute;
+            left: -3px;
+            bottom: -7px;
+            border-radius: 50%;
+            width: 0.5rem;
+            height: 0.5rem;
+            border: 0.125rem solid $product-switch-dnd-line-color;
+            background-color: transparent;
+        }
```

Comment:
You don't need this part. We have the "circle" only on the top. 
<img width="384" alt="Screen Shot 2019-10-21 at 4 36 43 PM" src="https://user-images.githubusercontent.com/39598672/67241199-fcdd4080-f420-11e9-8886-a35b33cb5163.png">